app/views/exam_authorization_requests/_pending.html.erb

```diff
@@ -1,5 +1,14 @@
-<%= t :exam_authorization_pending_explanation_html,
-      date: l(authorization_request.exam.start_time, format: :long),
-      end_time: l(authorization_request.exam_registration.end_time, format: :long),
-      location: Organization.current.time_zone,
-      edit_path: url_for(authorization_request.exam_registration) %>
+<% if authorization_request.exam_registration.ended? %>
+  <%= t :exam_authorization_pending_confirmation_soon_html %>
+<% else %>
```

Comment:
not sure if we have a general policy on it, but I find it a little odd nesting ifs instead of using elsif?

Reply:
Whoops, will rebase!